app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
It seems that we can grab all the cases we want without a class-based selector (see last commit)

`input:checked + label` doesn't apply to the counter-examples I mentioned, as they (coincidently) have a structure like `<label><input></label>`.

Making the distinction based on this structure difference appears super dirty though...